modules/platforms/dotnet/Apache.Ignite.Core.Tests/ProjectFilesTest.cs

```diff
@@ -87,6 +87,36 @@ public void TestCsprojOptimizeCode()
                 "Invalid optimize setting in release mode: ");
         }
 
+        /// <summary>
+        /// Tests that all public types have <see cref="object.ToString"/> method overridden.
+        /// </summary>
+        [Test]
+        public void TestPublicTypesHaveToStringOverride()
+        {
+            var csFiles = TestUtils.GetDotNetSourceDir().GetFiles("*.cs", SearchOption.AllDirectories);
```

Comment:
I wonder if we should narrow down to Apache.Ignite.Core project instead of all the projects in the dotnet directory because the test project is for Apache.Ignite.Core? I have made a change on that, but just wanna make sure before I commit it.